packages/spectacular/src/lib/bootstrap-test-application/bootstrap-test-application.ts

```diff
@@ -0,0 +1,25 @@
+import { TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { TestRootComponent, TestRootScam, testRootTagName } from '../test-root';
```

Comment:
Expand barrel imports to void production build issues.

Reply:
✅ Already did.